File: dataset/human/c/file_2629.c
#include<stdio.h>
int main(){
  char s[4];
  int a=0,b=0,c=0,i;
  scanf("%s",&s);
  for(i=0;i<3;i++){
    if(s[i]=='a' && a==0){
      a=1;}
   if(s[i]=='b' && b==0){
      b=1;}
    if(s[i]=='c' && c==0){
      c=1; }
  }
  if(a==1&&b==1&&c==1){
    printf("Yes");
  }
  else{
    printf("No");
  }
  return 0;
}

File: dataset/human/java/file_2629.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String s = sc.next();
		String t = sc.next();

		char[] sChar = s.toCharArray();
		char[] tChar = t.toCharArray();

		Arrays.sort(sChar);
		Arrays.sort(tChar);

		String sString = String.valueOf(sChar);
		StringBuffer sb = new StringBuffer(String.valueOf(tChar));
		String tString = sb.reverse().toString();

		System.out.println(sString.compareTo(tString) < 0 ? "Yes" : "No");
	}
}
